### PR TITLE
[Remove Stream/Optional usage] LPS-173789 module  bulk-selection-impl

### PR DESCRIPTION
**Pull request by JanisZhang**
Ticket: https://issues.liferay.com/browse/LPS-173789

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:bulk-editing

**Comment by Kvale1733** [comment]
@JanisZhang No regressions.

File: modules/apps/bulk/bulk-selection-impl/src/main/java/com/liferay/bulk/selection/internal/BulkSelectionBackgroundTaskExecutor.java [hunk 1]
```diff
@@ -71,62 +70,56 @@ public BackgroundTaskResult execute(BackgroundTask backgroundTask) {
...
-							_log.error(exception);
-						}
-					});
-			});
+		BulkSelectionAction<Object> bulkSelectionAction = _getService(
+			(Class<BulkSelectionAction<Object>>)
+				(Class<?>)BulkSelectionAction.class,
+			bulkSelectionActionClassName);
+
+		if (bulkSelectionAction != null) {
```
Comment (dantewang) [inline modules/apps/bulk/bulk-selection-impl/src/main/java/com/liferay/bulk/selection/internal/BulkSelectionBackgroundTaskExecutor.java:78]: @JanisZhang Please flip to buy space.

File: modules/apps/bulk/bulk-selection-impl/src/main/java/com/liferay/bulk/selection/internal/BulkSelectionBackgroundTaskExecutor.java [hunk 2]
```diff
@@ -71,62 +70,56 @@ public BackgroundTaskResult execute(BackgroundTask backgroundTask) {
...
+			String bulkSelectionFactoryClassName = (String)taskContextMap.get(
+				BulkSelectionBackgroundTaskConstants.
+					BULK_SELECTION_FACTORY_CLASS_NAME);
+
+			BulkSelectionFactory<?> bulkSelectionFactory = _getService(
+				(Class<BulkSelectionFactory<?>>)
+					(Class<?>)BulkSelectionFactory.class,
+				bulkSelectionFactoryClassName);
+
+			if (bulkSelectionFactory != null) {
```
Comment (dantewang) [inline modules/apps/bulk/bulk-selection-impl/src/main/java/com/liferay/bulk/selection/internal/BulkSelectionBackgroundTaskExecutor.java:93]: @JanisZhang Also here.

**Comment by dantewang** [comment]
@JanisZhang Please resend, thanks!

**Pull request closed without merging**